### PR TITLE
Hc/update version

### PR DESCRIPTION
**Pull request by raygun-harrison**
REQUIRED: Must merge RUM611 into the 'update-version' branch before this branch can be merged into master. 

Changes:
- Updates version numbers within the repo, needed before publishing to NPM

- All updates from RUM-613 and RUM-611 (see previous PRs)

### CODE REVIEW

File: demo/App.tsx [hunk 1]
```diff
@@ -140,13 +154,14 @@ const App = () => {
...
-                    console.log('Existing customData', data);
-                    return {
-                      replaceAllData: true
-                    };
-                  })
+                onPress={() => {
+                  const updater = (data: CustomData) => {
+                    // Do something with the custom data if you wish
+                    console.log("DATA:", data);
+                    return data
```
Comment (BenjaminHarding) [inline demo/App.tsx:161]: I think a real-use case would be good to include here

File: demo/App.tsx [hunk 2]
```diff
@@ -140,13 +154,14 @@ const App = () => {
...
+                  }
+                  RaygunClient.updateCustomData(updater)
```
Comment (BenjaminHarding) [inline demo/App.tsx:163]: Missing semi colons :)

File: demo/App.tsx [hunk 3]
```diff
@@ -160,10 +175,18 @@ const App = () => {
...
+                  const customData4: CustomData = {"Key_4": [42, 65] };
+                  const customData5: CustomData = {"Key_5": customData4};
+                  RaygunClient.addCustomData(customData1);
+                  RaygunClient.addCustomData(customData2);
+                  RaygunClient.addCustomData(customData3);
+                  RaygunClient.addCustomData(customData4);
+                  RaygunClient.addCustomData(customData5);
+                }
                 }
                 title="Add Random Custom Data"
```
Comment (BenjaminHarding) [inline demo/App.tsx:191]: This is no longer adding random custom data, so the title will need updating

File: demo/App.tsx [hunk 4]
```diff
@@ -195,7 +221,13 @@ const App = () => {
                 color="green"
                 testID="reInitializeBtn"
                 accessibilityLabel="reInitializeBtn"
-                onPress={() => reInitialize()}
+                onPress={() => {
+                  const options: RaygunClientOptions = {
+                    apiKey: 'notAValidKey',
+                    version: '42',
+                  }
```
Comment (BenjaminHarding) [inline demo/App.tsx:228]: Missing semi-colon :)

File: sdk/package.json [hunk 5]
```diff
@@ -16,14 +16,17 @@
...
     "build_sdk": "npm i && npm pack",
     "local_copy_sdk": "copyfiles *.tgz ../demo",
-    "build_demo": "cd ../demo && npm i",
+    "build_demo_win": "cd ../demo && npm uninstall -S raygun4reactnative && npm install raygun4reactnative-%npm_package_version%.tgz && npm i",
+    "build_demo_mac_lin": "cd ../demo && npm uninstall -S raygun4reactnative && npm install raygun4reactnative-$npm_package_version.tgz && npm i",
     "run_demo": "cd ../demo && npx react-native run-android",
-    "fully_automated": "npm run clean_dir_all && npm run build_sdk && npm run local_copy_sdk && npm run build_demo && npm run run_demo",
+    "build_no_run_win": "npm run clean_dir_all && npm run build_sdk && npm run local_copy_sdk && npm run build_demo_win",
+    "fully_automated_win": "npm run clean_dir_all && npm run build_sdk && npm run local_copy_sdk && npm run build_demo_win && npm run run_demo",
+    "fully_automated_mac_lin": "npm run clean_dir_all && npm run build_sdk && npm run local_copy_sdk && npm run build_demo_mac_lin && npm run run_demo",
```
Comment (BenjaminHarding) [inline sdk/package.json:29]: There is some duplicate code between these commands which we should clean up :)

**Review by BenjaminHarding — COMMENTED** [review]
Nice work.

File: demo/App.tsx (hunk 1 above)
Comment (raygun-harrison) [reply demo/App.tsx:161]: Adding in some functionality that remove numbers and arrays

DONE

File: demo/App.tsx (hunk 2 above)
Comment (raygun-harrison) [reply demo/App.tsx:163]: DONE

File: demo/App.tsx (hunk 3 above)
Comment (raygun-harrison) [reply demo/App.tsx:191]: DONE

File: demo/App.tsx (hunk 4 above)
Comment (raygun-harrison) [reply demo/App.tsx:228]: DONE

File: sdk/package.json (hunk 5 above)
Comment (raygun-harrison) [reply sdk/package.json:29]: DONE (arguably)

**Review by BenjaminHarding — APPROVED** [review]
👍

**Pull request merged**